API-Samples/Python/VideoIndexerClient/VideoIndexerClient.py

```diff
@@ -398,3 +398,70 @@ def get_player_widget_url_async(self, video_id:str) -> None:
 
         url = response.url
         print(f'Got the player widget URL: {url}')
+
+    def get_textual_summary(self, video_id:str, deployment_name:str, lenght:str, style:str, timeout_sec:Optional[int]=None) -> None:
+        '''
+        Calls the TextualSummarization API. First a call is made with the POST method, to create the video summary:
+        (https://api-portal.videoindexer.ai/api-details#api=Operations&operation=Create-Video-Summary)
+        Then, a call is made with the get method to retrieve the video summary and the function returns it.
+        (https://api-portal.videoindexer.ai/api-details#api=Operations&operation=Get-Video-Summary)
+
+        :param video_id: The video ID
+        :param deploymentName: The name of the deployment
+        :param length: The length of the summary

```

Comment:
Please specify the allowed values for `length` and `style` in the docstring and at the function start - length
The length of the summary. Allowed values: Medium / Short / Long
style- The style of the summary. Allowed values: Neutral / Casual / Formal

Reply:
understood, changed it